project/plugins.sbt

```diff
@@ -5,3 +5,5 @@ addSbtPlugin("com.github.sbt" % "sbt-ci-release" % "1.5.10")
 addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.6")
 
 addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.33")
+
+libraryDependencies += "org.scala-js" %% "scalajs-env-jsdom-nodejs" % "1.1.0"
```

Comment:
How come this needs to be a dependency of the metabuild?

Reply:
It's used by the ScalaJS plugin to run the emitted JS. It's not actually needed as a compile time dependency.

Reply:
In fact, here it is being used in the `build.sbt` :)
https://github.com/scoverage/scalac-scoverage-plugin/blob/67fb9040fc0166c26b485b937057aa2d04d5cf22/build.sbt#L129-L137

Reply:
Ah of course, missed that. Thanks!